### PR TITLE
[Cache] Document the different cache status behavior after a purge

### PR DESCRIPTION
**Pull request by ngayerie**
PCX-9851

### CODE REVIEW

File: content/cache/how-to/purge-cache/purge-everything.md [hunk 1]
```diff
@@ -13,4 +13,9 @@ Purging everything immediately clears all resources from your CDN cache in all C
 1. Log in to your [Cloudflare dashboard](https://dash.cloudflare.com/login), and select your account and domain.
 2. Select **Caching** > **Configuration**.
 3. Under **Purge Cache**, select **Purge Everything**. A warning window appears.
-4. If you agree, select **Purge Everything**.
\ No newline at end of file
+4. If you agree, select **Purge Everything**.
+
```
Comment (angelampcosta) [inline content/cache/how-to/purge-cache/purge-everything.md:17]:
```suggestion
```

File: content/cache/how-to/purge-cache/purge-by-hostname.md [hunk 2]
```diff
@@ -23,4 +23,11 @@ Purging by hostname means that all assets at URLs with a host that matches one o
 
 You can purge hostnames via the Cloudflare API. For more information, refer to the [API documentation](/api/operations/zone-purge). You can use up to 30 hostnames per API call and make up to 30,000 purge API calls in a 24-hour period.
 
-{{</Aside>}}
\ No newline at end of file
+{{</Aside>}}
+
+## Resulting cache status
+
+Purging by hostname deletes the resource: for the next request the [CF-Cache-Status](/cache/concepts/default-cache-behavior/#cloudflare-cache-responses) header will be `MISS`.
```
Comment (angelampcosta) [inline content/cache/how-to/purge-cache/purge-by-hostname.md]:
```suggestion
Purging by hostname deletes the resource, resulting in the CF-Cache-Status header being set to [`MISS`](/cache/concepts/cache-responses/#miss) for subsequent requests.
```

File: content/cache/how-to/purge-cache/purge-by-hostname.md [hunk 3]
```diff
@@ -23,4 +23,11 @@ Purging by hostname means that all assets at URLs with a host that matches one o
...
+
+If [tiered cache](/cache/how-to/tiered-cache/) is used, purging by hostname may return `EXPIRED` because lower tier tries to revalidate with upper tier to reduce load on upper tier.
+Depending on whether the upper tier has the resource or not, and whether the eyeball is reaching the lower tier or the upper tier, `EXPIRED` or `MISS` are returned.
```
Comment (angelampcosta) [inline content/cache/how-to/purge-cache/purge-by-hostname.md]:
```suggestion
Depending on whether the upper tier has the resource or not, and whether the end user is reaching the lower tier or the upper tier, `EXPIRED` or `MISS` are returned.
```

File: content/cache/how-to/purge-cache/purge-by-hostname.md [hunk 4]
```diff
@@ -23,4 +23,11 @@ Purging by hostname means that all assets at URLs with a host that matches one o
...
 
-{{</Aside>}}
\ No newline at end of file
+{{</Aside>}}
+
+## Resulting cache status
+
+Purging by hostname deletes the resource: for the next request the [CF-Cache-Status](/cache/concepts/default-cache-behavior/#cloudflare-cache-responses) header will be `MISS`.
+
+If [tiered cache](/cache/how-to/tiered-cache/) is used, purging by hostname may return `EXPIRED` because lower tier tries to revalidate with upper tier to reduce load on upper tier.
```
Comment (angelampcosta) [inline content/cache/how-to/purge-cache/purge-by-hostname.md]:
```suggestion
If [tiered cache](/cache/how-to/tiered-cache/) is used, purging by hostname may return `EXPIRED`, as the lower tier tries to revalidate with the upper tier to reduce load on the latter.
```

File: content/cache/how-to/purge-cache/purge-by-single-file.md [hunk 5]
```diff
@@ -49,4 +49,8 @@ For information on how to use single-file purge to purge assets cached by a Work
 
 If you have a [Transform Rule](/rules/transform/) in place that is modifying part of a URL path, you must use the non-transform (end user) URL when performing single file purge so that purge can take effect.
 
-{{</Aside>}}
\ No newline at end of file
+{{</Aside>}}
+
+## Resulting cache status
+
+Single-file purge deletes the resource: for the next request the [CF-Cache-Status](/cache/concepts/default-cache-behavior/#cloudflare-cache-responses) header will be `MISS`.
```
Comment (angelampcosta) [inline content/cache/how-to/purge-cache/purge-by-single-file.md]:
```suggestion
Purging by single-file deletes the resource, resulting in the CF-Cache-Status header being set to [`MISS`](/cache/concepts/cache-responses/#miss) for subsequent requests.
```

File: content/cache/how-to/purge-cache/purge-by-tags.md [hunk 6]
```diff
@@ -56,3 +56,10 @@ When your content reaches our edge network, Cloudflare:
 You can purge using cache-tags via the Cloudflare API. For more information, refer to the [API documentation](/api/operations/zone-purge). You can purge up to 30 cache-tags per API call and up to 250,000 cache-tags per a 24-hour period.
 
 {{</Aside>}}
+
+## Resulting cache status
+
+Purging by tag deletes the resource: for the next request the [CF-Cache-Status](/cache/concepts/default-cache-behavior/#cloudflare-cache-responses) header will be `MISS`.
```
Comment (angelampcosta) [inline content/cache/how-to/purge-cache/purge-by-tags.md]:
```suggestion
Purging by tag deletes the resource, resulting in the CF-Cache-Status header being set to [`MISS`](/cache/concepts/cache-responses/#miss) for subsequent requests.
```

File: content/cache/how-to/purge-cache/purge-by-tags.md [hunk 7]
```diff
@@ -56,3 +56,10 @@ When your content reaches our edge network, Cloudflare:
...
+
+If [tiered cache](/cache/how-to/tiered-cache/) is used, purging by tag may return `EXPIRED` because lower tier tries to revalidate with upper tier to reduce load on upper tier.
```
Comment (angelampcosta) [inline content/cache/how-to/purge-cache/purge-by-tags.md]:
```suggestion
If [tiered cache](/cache/how-to/tiered-cache/) is used, purging by tag may return `EXPIRED`, as the lower tier tries to revalidate with the upper tier to reduce load on the latter.
```

File: content/cache/how-to/purge-cache/purge-by-tags.md [hunk 8]
```diff
@@ -56,3 +56,10 @@ When your content reaches our edge network, Cloudflare:
...
+Depending on whether the upper tier has the resource or not, and whether the eyeball is reaching the lower tier or the upper tier, `EXPIRED` or `MISS` are returned.
```
Comment (angelampcosta) [inline content/cache/how-to/purge-cache/purge-by-tags.md]:
```suggestion
Depending on whether the upper tier has the resource or not, and whether the end user is reaching the lower tier or the upper tier, `EXPIRED` or `MISS` are returned.
```

File: content/cache/how-to/purge-cache/purge-everything.md [hunk 9]
```diff
@@ -13,4 +13,9 @@ Purging everything immediately clears all resources from your CDN cache in all C
...
+
+## Resulting cache status
+
+Purge Everything invalidates the resource: for the next request the [CF-Cache-Status](/cache/concepts/default-cache-behavior/#cloudflare-cache-responses) header will be `EXPIRED`.
```
Comment (angelampcosta) [inline content/cache/how-to/purge-cache/purge-everything.md]:
```suggestion
Purge Everything invalidates the resource, resulting in the CF-Cache-Status header indicating [`EXPIRED`](/cache/concepts/cache-responses/#expired) for subsequent requests.
```

File: content/cache/how-to/purge-cache/purge_by_prefix.md [hunk 10]
```diff
@@ -42,6 +42,13 @@ If you have a [Transform Rule](/rules/transform/) in place that is modifying par
 
 {{</Aside>}}
 
+## Resulting cache status
+
+Purging by prefix deletes the resource: for the next request the [CF-Cache-Status](/cache/concepts/default-cache-behavior/#cloudflare-cache-responses) header will be `MISS`.
```
Comment (angelampcosta) [inline content/cache/how-to/purge-cache/purge_by_prefix.md]:
```suggestion
Purging by prefix deletes the resource, causing CF-Cache-Status header to show [`MISS`](/cache/concepts/cache-responses/#miss) for the subsequent request.
```

File: content/cache/how-to/purge-cache/purge_by_prefix.md [hunk 11]
```diff
@@ -42,6 +42,13 @@ If you have a [Transform Rule](/rules/transform/) in place that is modifying par
...
+
+If [tiered cache](/cache/how-to/tiered-cache/) is used, purging by prefix may return `EXPIRED` because lower tier tries to revalidate with upper tier to reduce load on upper tier.
```
Comment (angelampcosta) [inline content/cache/how-to/purge-cache/purge_by_prefix.md]:
```suggestion
If [tiered cache](/cache/how-to/tiered-cache/) is used, purging by prefix may return `EXPIRED`, as the lower tier tries to revalidate with the upper tier to reduce load on the latter.
```

File: content/cache/how-to/purge-cache/purge_by_prefix.md [hunk 12]
```diff
@@ -42,6 +42,13 @@ If you have a [Transform Rule](/rules/transform/) in place that is modifying par
...
+Depending on whether the upper tier has the resource or not, and whether the eyeball is reaching the lower tier or the upper tier, `EXPIRED` or `MISS` are returned.
```
Comment (angelampcosta) [inline content/cache/how-to/purge-cache/purge_by_prefix.md]:
```suggestion
Depending on whether the upper tier has the resource or not, and whether the end user is reaching the lower tier or the upper tier, `EXPIRED` or `MISS` are returned.
```

**Comment by angelampcosta** [comment]
@ngayerie thank for opening the PR. I made a few suggestions. Let me know if they make sense.

**Comment by ngayerie** [comment]
Thanks @angelampcosta ! That's great!

File: content/cache/how-to/purge-cache/purge-by-hostname.md [hunk 13]
```diff
@@ -23,4 +23,11 @@ Purging by hostname means that all assets at URLs with a host that matches one o
 
 You can purge hostnames via the Cloudflare API. For more information, refer to the [API documentation](/api/operations/zone-purge). You can use up to 30 hostnames per API call and make up to 30,000 purge API calls in a 24-hour period.
 
-{{</Aside>}}
\ No newline at end of file
+{{</Aside>}}
+
+## Resulting cache status
+
+Purging by hostname deletes the resource, resulting in the CF-Cache-Status header being set to [`MISS`](/cache/concepts/cache-responses/#miss) for subsequent requests.
```
Comment (angelampcosta) [inline content/cache/how-to/purge-cache/purge-by-hostname.md]:
```suggestion
Purging by hostname deletes the resource, resulting in the `CF-Cache-Status` header being set to [`MISS`](/cache/concepts/cache-responses/#miss) for subsequent requests.
```

File: content/cache/how-to/purge-cache/purge-by-single-file.md [hunk 14]
```diff
@@ -49,4 +49,8 @@ For information on how to use single-file purge to purge assets cached by a Work
 
 If you have a [Transform Rule](/rules/transform/) in place that is modifying part of a URL path, you must use the non-transform (end user) URL when performing single file purge so that purge can take effect.
 
-{{</Aside>}}
\ No newline at end of file
+{{</Aside>}}
+
+## Resulting cache status
+
+Purging by single-file deletes the resource, resulting in the CF-Cache-Status header being set to [`MISS`](/cache/concepts/cache-responses/#miss) for subsequent requests.
```
Comment (angelampcosta) [inline content/cache/how-to/purge-cache/purge-by-single-file.md]:
```suggestion
Purging by single-file deletes the resource, resulting in the `CF-Cache-Status` header being set to [`MISS`](/cache/concepts/cache-responses/#miss) for subsequent requests.
```

File: content/cache/how-to/purge-cache/purge-by-tags.md [hunk 15]
```diff
@@ -56,3 +56,10 @@ When your content reaches our edge network, Cloudflare:
 You can purge using cache-tags via the Cloudflare API. For more information, refer to the [API documentation](/api/operations/zone-purge). You can purge up to 30 cache-tags per API call and up to 250,000 cache-tags per a 24-hour period.
 
 {{</Aside>}}
+
+## Resulting cache status
+
+Purging by tag deletes the resource, resulting in the CF-Cache-Status header being set to [`MISS`](/cache/concepts/cache-responses/#miss) for subsequent requests.
```
Comment (angelampcosta) [inline content/cache/how-to/purge-cache/purge-by-tags.md]:
```suggestion
Purging by tag deletes the resource, resulting in the `CF-Cache-Status` header being set to [`MISS`](/cache/concepts/cache-responses/#miss) for subsequent requests.
```

File: content/cache/how-to/purge-cache/purge-everything.md [hunk 16]
```diff
@@ -13,4 +13,8 @@ Purging everything immediately clears all resources from your CDN cache in all C
 1. Log in to your [Cloudflare dashboard](https://dash.cloudflare.com/login), and select your account and domain.
 2. Select **Caching** > **Configuration**.
 3. Under **Purge Cache**, select **Purge Everything**. A warning window appears.
-4. If you agree, select **Purge Everything**.
\ No newline at end of file
+4. If you agree, select **Purge Everything**.
+
+## Resulting cache status
+
+Purge Everything invalidates the resource, resulting in the CF-Cache-Status header indicating [`EXPIRED`](/cache/concepts/cache-responses/#expired) for subsequent requests.
```
Comment (angelampcosta) [inline content/cache/how-to/purge-cache/purge-everything.md]:
```suggestion
Purge Everything invalidates the resource, resulting in the `CF-Cache-Status` header indicating [`EXPIRED`](/cache/concepts/cache-responses/#expired) for subsequent requests.
```

File: content/cache/how-to/purge-cache/purge_by_prefix.md [hunk 17]
```diff
@@ -42,6 +42,13 @@ If you have a [Transform Rule](/rules/transform/) in place that is modifying par
 
 {{</Aside>}}
 
+## Resulting cache status
+
+Purging by prefix deletes the resource, causing CF-Cache-Status header to show [`MISS`](/cache/concepts/cache-responses/#miss) for the subsequent request.
```
Comment (angelampcosta) [inline content/cache/how-to/purge-cache/purge_by_prefix.md]:
```suggestion
Purging by prefix deletes the resource, causing `CF-Cache-Status` header to show [`MISS`](/cache/concepts/cache-responses/#miss) for the subsequent request.
```

**Review by angelampcosta — APPROVED** [review]

**Comment by angelampcosta** [comment]
@ngayerie please don't merge. We need a final review from Zaidoon.

**Pull request merged**